src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -19,6 +19,7 @@ public class HttpsTests : ConditionalWcfTest
     // Server: BasicHttpsBinding default value is Soap11
     [ConditionalFact(nameof(Root_Certificate_Installed))]
     [OuterLoop]
+    [ActiveIssue(1123, PlatformID.AnyUnix)]
```

Comment:
This should not be an ActiveIssue. The product is okay. It's the setup condition is not met that causes the test failure on AnyUnix. This test should be disabled on AnyUnix by stronger condition.